lib/ajax.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/*
+Plugin Name: Genesis Simple Love
+Plugin URI: https://wordpress.org/plugins/genesis-simple-love
+Description: Add Love feature on Genesis Simple Share Plugin. <strong>This plugin requires Genesis Simple Share. Thanks!</strong>
+Version: 1.0
+Author: phpbits
+Author URI: http://codecanyon.net/user/phpbits/portfolio?ref=phpbits
+License: GPL2
+*/
+if(!class_exists('PHPBITS_GenesisSimpleLove')):
```

Comment:
spacing needs updated and preference for `}` over `: endif;`

~~~php
if ( ! class_exists( 'PHPBITS_GenesisSimpleLove' ) ) {
~~~